03 - Showcases & Templates/Plugin Showcases/Breadcrumbs for Comparative Law.md

```diff
@@ -9,9 +9,11 @@ publish: true
 
 *Note: this is a refined version of the post by [[lkadre]] on [this discussion](https://github.com/SkepticMystic/breadcrumbs/discussions/175) on Github about [[How to get the most out of the Breadcrumbs plugin]].*
 
-#### Example: Concept notes (Law)
+Since the [[breadcrumbs|Breadcrumbs]] plugin is so powerful, it can be useful to see an showcase for how other people use it to maximise creativity and analysis. 
+
+This is specifically about my notes on Law, which I acknowledge is probably a very niche topic but hopefully it can give you some ideas of how to set up your hierarchies and especially how to work with notes in multiple hierarchies. You don't have to choose one!
 
-This is specifically about my notes on Law, which I acknowledge is probably very niche but hopefully it can give you some ideas of how to set up your hierarchies and especially how to work with notes in multiple hierarchies. You don't have to choose one!
+#### Example: Concept notes (Law)
```

Comment:
I don't really get the purpose of this heading. To me, it does not make sense to have a subheading, when there is no other subheading 😅 

I'd suggest either removing the subheading or adding a second one somewhere.

Reply:
I don't mind it for navigation purposes. Click on the TOC on the right to jump to the example

Reply:
ah, haven't thought of that use case. makes sense!